### PR TITLE
PXP-8292 - Updating documentation on workspaces

### PR DESCRIPTION
**Pull request by xritter1**
Jira Ticket: https://ctds-planx.atlassian.net/browse/PXP-8292

### Improvements
Adding more documentation about workspaces.

### CODE REVIEW

**Review by qiongl-CTDS — APPROVED** [review]
LGTM

**Review by cgmeyer — APPROVED** [review]

File: content/resources/user/analyze-data.md [hunk 1]
```diff
@@ -171,21 +184,28 @@ git pull origin master
 ```
 
 ### Examples
-1) Most requests sent to a Gen3 data commons API will require an authorization token to be sent in the request's header. The SDK class _Gen3Auth_ is used for authentication purposes, and has functions for generating these access tokens. The notebook ["Gen3_authentication.ipynb"](#5-jupyter-notebook-demos) can be uploaded to a workspace and run to demonstrate how to initialize an instance of the Gen3Auth class.
+1.) Most requests sent to a Gen3 data commons API will require an authorization token to be sent in the request's header. The SDK class _Gen3Auth_ is used for authentication purposes, and has functions for generating these access tokens. Users do need to authenticate when using the SDK from the terminal, but do not need to authenticate once being logged in and working in the workspace of a Data Commons.
+
+From the terminal run the following:
 
 ```
+git clone https://github.com/uc-cdis/gen3sdk-python.git
```
Comment (svburke) [inline content/resources/user/analyze-data.md:192]:
```suggestion
!git clone https://github.com/uc-cdis/gen3sdk-python.git
```

**Review by svburke — APPROVED** [review]

**Review by qiongl-CTDS — APPROVED** [review]

File: content/resources/user/submit-data.md [hunk 2]
```diff
@@ -39,6 +39,12 @@ Do the following to create your first `core_metadata_collection` record:
 2. Click on 'Submit Data'
 3. Find your project in the List of Projects and click 'Submit Data'
```
Comment (gkuffel) [inline content/resources/user/submit-data.md:40]: "List" shouldn't be capitalized.

File: content/resources/user/submit-data.md [hunk 3]
```diff
@@ -39,6 +39,12 @@ Do the following to create your first `core_metadata_collection` record:
...
 4. Click 'Use Form Submission' and choose `core_metadata_collection` from the dropdown list (or [edit and upload this TSV](gen3_core_metadata_collection_template.tsv) by clicking 'Upload File' then 'Submit')
+
+    ![node_dropdown.png](node_dropdown.png)
+
+    ![cmc_form.png](cmc_form.png)
+
+
 5. Fill in the required information (see Note below)
```
Comment (gkuffel) [inline content/resources/user/submit-data.md:48]: "Note should not be capitalized"

**Review by gkuffel — CHANGES_REQUESTED** [review]

**Review by qiongl-CTDS — APPROVED** [review]
lgtm

**Review by svburke — APPROVED** [review]

**Pull request merged**